applications/RomApplication/custom_strategies/rom_builder_and_solver.h

```diff
@@ -329,13 +195,11 @@ class ROMBuilderAndSolver : public BuilderAndSolver<TSparseSpace, TDenseSpace, T
     {
         auto& r_dof_set = BaseType::GetDofSet();
         BaseType::mEquationSystemSize = r_dof_set.size();
-        int ndofs = static_cast<int>(r_dof_set.size());
-
-        #pragma omp parallel for firstprivate(ndofs)
-        for (int i = 0; i < ndofs; i++){
-            auto dof_iterator = r_dof_set.begin() + i;
-            dof_iterator->SetEquationId(i);
-        }
+        IndexPartition<IndexType>(r_dof_set.size()).for_each([&](IndexType Index)

```

Comment:
```suggestion
        IndexPartition<IndexType>(r_dof_set.size()).for_each([](IndexType Index)
```
(very minor) I think you need to capture nothing in here.

Reply:
I'm capturing `r_dof_set`.

Reply:
Ah sure, for the size. Forget about my comment.